keyboards/gray_studio/bd60/info.json

```diff
@@ -0,0 +1,121 @@
+{
+    "keyboard_name": "BD60",
+    "url": "",
+    "maintainer": "edwardslau",
+    "manufacturer":"GrayStudio",
+    "processor": "STM32F072",
+    "diode_direction": "COL2ROW",
+    "bootloader": "stm32-dfu",
+       "usb": {
+        "vid":"0x4753",
+        "pid":"0x4060",
+        "device_version": "1.0.0"
+    },
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": true,
+        "rgblight": true
+    },
+    "ws2812": {
+        "pin": "B10"
+    },
+    "matrix_pins": {
+        "cols": ["A3", "C14", "C13", "B9", "B8", "B7", "B6", "B5", "B4", "B3", "A15", "B0", "A8", "B15", "B14"],
+        "rows": ["B13", "B12", "A2", "A0", "A1"]
+    },
+    "rgblight": {
+        "led_count": 1,
+        "sleep": true,
+        "max_brightness": 150,
+        "animations": {
+            "alternating": true,
+            "breathing": true,
+            "christmas": true,
+            "knight": true,
+            "rainbow_mood": true,
+            "rainbow_swirl": true,
+            "rgb_test": true,
+            "snake": true,
+            "static_gradient": true,
+            "twinkle": true
+        }
+    },
+    "community_layouts": ["60_ansi_shift_bs_rshift"],
```

Comment:
Could the following Community Layouts (`layouts/default`) be added?:
- 60_ansi
- 60_ansi_tsangan
- 60_tsangan_hhkb
- 60_hhkb

or alternatively, a `matrix_diagram.md` file provided.

Reply:
Definitely!

Reply:
And also can you help me out on the check fail? It says that claiming the community layout does not exist

Reply:
> And also can you help me out on the check fail? It says that claiming the community layout does not exist

my fault lol! rectified `_shift_` to `_split_` in a suggestion

Reply:
Thank you sooooo much!

Reply:
if you need help with the requested Community Layouts, please do give me a nudge on here 🙂

Reply:
Do I need to manually add those layouts you mentioned in the files? @dunk2k

Reply:
Each layout would be added as an entry in `layouts`, of `info.json`. Currently '60_ansi_split_bs_rshift` is an entry.
i.e.
```
"layouts" : {
    "60_ansi": {
        "layout": [
            ...
        ]
    },
    "60_ansi_split_bs_rshift": {
        "layout": [
            ...
        ]
    },
},
```
and `community_layouts` array, in `info.json` appended

Reply:
Thank you!